### PR TITLE
fix(tslib): standarised to use private _ over # to prevent:

### PR DESCRIPTION
**Pull request by Ansonhkg**
Replacement for this:
https://github.com/LIT-Protocol/js-sdk/pull/495

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
 
 
+"@aashutoshrathi/word-wrap@^1.2.3":
```
Comment (joshLong145) [inline yarn.lock:5]: Is there a reason we need to update the lock with this pr? I dont see any changes to our `package.json` so just curious.

**Review by FedericoAmura — APPROVED** [review]
There is one more # call in [DiscordProvider](https://github.com/LIT-Protocol/js-sdk/blob/master/packages/lit-auth-client/src/lib/providers/DiscordProvider.ts#L210) 

Then our sdk would be #-calls free =D

**Review by joshLong145 — COMMENTED** [review]
One small comment but looks good

File: yarn.lock (hunk 1 above)
Comment (Ansonhkg) [reply yarn.lock:5]: `package.json` file is updated in the `master` branch but the `yarn.lock` is old, and i `checkout -b` from `master` that's why `yarn.lock` is added here.

**Review by jtary — APPROVED** [review]
LGTM

**Pull request merged**